src/utils/utilities.js

```diff
@@ -0,0 +1,22 @@
+var percent = function(wins, losses){
```

Comment:
With percent  I was thinking its is a universal concept so I would keep all game/app/context specific logic out of it and just have a pure % function. We don't need to handle NaN as JS will do that for us.

```js
const percent = function(portion, total) {
  return Math.round(portion / total * 100);
}

percent(10, 100) // 10
percent(10,'100') // 10 - '100' typecast to int
percent(10, 20.323) // 49
percent('a string', 10) // NaN - JS handles NaN for us
percent(false, 100) // 0 - false typcast to 0
percent(100, false) //Infinity - false typecast to 0
percent() //NaN
```
All our logic is better in percentOfPlayerWins. I've put a few checks in which covers most 'possible' results... negative values make it through incorrectly & false is typecast to 0. In this instance I'm not too worried about data validation or throwing a meaningful error as our data is all machine driven so there's not really much chance of getting badly formatted data. The only value we do need to account for is `undefined` which could be passed to percent() and will result in NaN which is handled by !!winPercent which casts as false. The isFinite check is probably overkill but doesn't hurt either.
```js
const percentOfPlayerWins = function(wins, losses) {
  let total = wins + losses;
  let winPercent = percent(wins, total); // returns number, NaN or Infinity/-Infinity
  return total > 9 && !!winPercent && isFinite(winPercent) ? winPercent + '%' : '-'; // Ensures we have a sensible result
}

percentOfPlayerWins(20, 30); // "40%"
percentOfPlayerWins(undefined, 100); // "-"
percentOfPlayerWins("asdasd", 100); // "-"
percentOfPlayerWins(false, 100); // "-"
percentOfPlayerWins(Infinity,100); // "-"
percentOfPlayerWins(100, false); // "100%" - kindof an issue
percentOfPlayerWins(100, 0); // "100%"
percentOfPlayerWins(100, -0); // "100%"  - kindof an issue
percentOfPlayerWins(-100, -0); // "-"  - kindof an issue
percentOfPlayerWins(-100, 100); // "-"
percentOfPlayerWins(-100, 10); // "-"
percentOfPlayerWins(); // "-"
```